### PR TITLE
Fake world dashboard backend

### PR DESCRIPTION
**Pull request by aszlam**
# Description

Moves fake world to lowlevel/minecraft in preparation for a standardized pyworld agent (which could be used in dashboard, for testing, and for simple pure-python gridworld experiments)

also adds a skeleton for e2e testing with verifiers

## Type of change

Please check the options that are relevant.

- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] Proposes a change (non-breaking change that isn't necessarily a bug)
- [x] Refactor
- [x] New feature (non-breaking change that adds a new functionality)
- [ ] Breaking change (fix or feature that would break some existing functionality downstream)
- [ ] This is a unit test
- [ ] Documentation only change
- [ ] Datasets Release
- [ ] Models Release

## Type of requested review

- [x] I want a thorough review of the implementation.
- [ ] I want a high level review. 
- [ ] I want a deep design review.

# Testing

Please describe the tests that you ran to verify your changes and how we can reproduce.

# Checklist:

- [x] I have performed manual end-to-end testing of the feature in my environment.
- [ ] I have added Docstrings and comments to the code.
- [ ] I have made changes to existing documentation where needed.
- [ ] I have added tests that show that the PR is functional.
- [x] New and existing unit tests pass locally with my changes.
- [ ] I have added relevant collaborators to review the PR before merge.
- [ ] [Polymetis only] I ran on hardware (1) that all scripts in `tests/scripts`, (2) asv benchmarks.

### CODE REVIEW

File: droidlet/lowlevel/minecraft/pyworld/physical_interfaces.py [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+        self.agent.world_interaction_occurred = True
+        if angle == 90:
+            self.agent.turn_left()
+        elif angle == -90:
+            self.agent.turn_right()
+        else:
+            raise ValueError("bad angle={}".format(angle))
+
+
+# FIXME!
```
Comment (kavyasrinet) [inline droidlet/lowlevel/minecraft/pyworld/physical_interfaces.py:151]: not sure I understand what needs to be fixed here ?

**Review by kavyasrinet — APPROVED** [review]
LGTM, thanks for adding this. I think the current craftassist tests will also test this path change

**Review by snyxan — APPROVED** [review]
LGTM

File: droidlet/lowlevel/minecraft/pyworld/physical_interfaces.py (hunk 1 above)
Comment (aszlam) [reply droidlet/lowlevel/minecraft/pyworld/physical_interfaces.py:151]: its an old fixme, just copied over.   but we aren't using turn_left and turn_right anywhere, I think these are dead now; the fixme was to investigate and torch

**Pull request merged**